hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/table/action/commit/FlinkDeletePartitionCommitActionExecutor.java

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.hudi.table.action.commit;
 
+import java.util.ArrayList;
```

Comment:
Please move after line 42 and follow the import check style.

Reply:
Done!